source/docs/variables.md

```diff
@@ -54,9 +54,9 @@ Variable | Description | Type
 
 Variable | Description | Type
 --- | --- | ---
-`page.published` | True if the post is not a draft | `boolean`
-`page.categories` | All categories of the post | `array` of ???
-`page.tags` | All tags of the post | `array` of ???
+`post.published` | True if the post is not a draft | `boolean`
+`post.categories` | All categories of the post | `array` of ???
+`post.tags` | All tags of the post | `array` of ???
```

Comment:
Those variables mentioned here is used in template when the `layout` is `post` and should access via `page.`.

Reply:
@SukkaW ok so this is the right fix #1273

Reply:
leme know so I make the change in other languages too

Reply:
For the theme developers, they may need to reuse the same article template for both home page and post page. So, a post variable will be used, in home page `post = page.posts.toArray()[0]`, in post page `post = page`.